.docker/Dockerfile

```diff
@@ -13,45 +13,50 @@ RUN apt-get update && \
       gcc \
       git \
       intltool \
-      libglib2.0-dev \
-      libxml2-dev \
-      meson \
+      python3-pip \
+      ninja-build \
       pkg-config \
       unzip \
       wget \
+      libxml2-dev libglib2.0-dev cmake libusb-1.0-0-dev gobject-introspection \
+                 libgtk-3-dev gtk-doc-tools  xsltproc libgstreamer1.0-dev \
+                 libgstreamer-plugins-base1.0-dev libgstreamer-plugins-good1.0-dev \
+                 libgirepository1.0-dev gettext \
     && rm -rf /var/lib/apt/lists/*
 
-RUN wget http://ftp.acc.umu.se/pub/GNOME/sources/aravis/0.6/aravis-0.6.4.tar.xz && \
-    tar xfJ aravis-0.6.4.tar.xz && \
-    rm aravis-0.6.4.tar.xz && \
-    cd aravis-0.6.4 && \
-    ./configure && \
-    make && \
-    checkinstall --pkgname aravis --requires="libglib2.0-dev, libxml2-dev" && \
-    ldconfig
+RUN pip3 install meson
+
+RUN wget https://github.com/AravisProject/aravis/releases/download/0.8.30/aravis-0.8.30.tar.xz -O aravis-0.8.30.tar.xz &&\
+    tar xfJ aravis-0.8.30.tar.xz && \
+    rm aravis-0.8.30.tar.xz && \
+    cd aravis-0.8.30 && \
+    meson setup build && \
+    cd build && \
+    ninja && \
+    checkinstall --pkgname aravis --pkgversion 0.8.30 ninja install
```

Comment:
Why did you remove the `--requires` tag? That makes sure the package manager installs those if not installed on your system.

Reply:
This originally was using the configure install. But aravis repo has updated the build instruction to use ninja build. Will add the